apps/crn-server/test/data-providers/contentful/research-output.data-provider.test.ts

```diff
@@ -1643,6 +1648,20 @@ describe('Research Outputs Data Provider', () => {
   });
 });
 
+describe('mapOutputVersions ', () => {
```

Comment:
i think you should add one test to just show how this mapping function really works - basic input and basic output, what do you think?